driver/modern_bpf/helpers/base/common.h

```diff
@@ -21,6 +21,8 @@
  * to the others `PT_REGS_PARAM...`
  */
 
+#ifndef PT_REGS_PARM6_CORE_SYSCALL
```

Comment:
Keep them when building with system libbpf that is < 1.2.0 thus does not contain the change.

Reply:
That is the case in some of our CI jobs.